code/modules/mob/living/carbon/human/species_types/ethari.dm

```diff
@@ -26,7 +26,9 @@
 	Following a major PR disaster Nanotrasen crews are no longer allowed to make Ethari crewmembers fetch the nuclear authentication disk or call them \"good boy\"."
 
 /datum/species/ethari/after_equip_job(datum/job/J, mob/living/carbon/human/H)
+	H.remove_language(/datum/language/common)
```

Comment:
Is this the best way to do this? Probably not. How else can I do it? I dunno. You tell me.

Reply:
To my knowledge, this is how you _have_ to do it.